packages/sdk/src/pda.ts

```diff
@@ -0,0 +1,9 @@
+import { PublicKey } from '@solana/web3.js';
+import { PREFIX, PROGRAM_ID } from './mpl-token-auth-rules';
+
+export const findRuleSetPDA = async (payer: PublicKey, name: string) => {
```

Comment:
Novice question: Does this need to be imported in like a global `index.ts` for it to be picked up as something importable in the npm package?

Reply:
Probably? I can add that.